resource/reactivedrop_german.txt

```diff
@@ -643,14 +643,14 @@
 "[english]asw_no_ammo"		"- NO AMMO -"
 "asw_low_ammo"		"- WENIG MUNITION -"
 "[english]asw_low_ammo"		"- LOW AMMO -"
-		"asw_alert_infested"		"- INFESTED -"
-		"[english]asw_alert_infested"		"- INFESTED -"
-		"asw_alert_poisoned"		"- POSIONED -"
-		"[english]asw_alert_poisoned"		"- POSIONED -"
-		"asw_alert_reloading"		"- RELOADING -"
-		"[english]asw_alert_reloading"		"- RELOADING -"
-		"asw_alert_ffguard"		"- FRIENDLY FIRE GUARD -"
-		"[english]asw_alert_ffguard"		"- FRIENDLY FIRE GUARD -"
+"asw_alert_infested"		"- INFESTED -"
+"[english]asw_alert_infested"		"- INFESTED -"
```

Comment:
Strings that are not translated yet should be indented with two tabs to help them stand out.